pennylane/ops/op_math/adjoint.py

```diff
@@ -160,11 +159,9 @@ def loop_fn(i):
 
     """
     if active_jit := compiler.active_compiler():
-        if lazy is False:
-            raise CompileError("Setting lazy=False is not supported with qjit.")
         available_eps = compiler.AvailableCompilers.names_entrypoints
         ops_loader = available_eps[active_jit]["ops"].load()
-        return ops_loader.adjoint(fn)
+        return ops_loader.adjoint(fn, laze=lazy)
```

Comment:
Wait, should this be `laze` or `lazy` ? I cannot see a `laze` option anywhere

Reply:
```suggestion
        return ops_loader.adjoint(fn, lazy=lazy)
```